### ISSUE TITLE
[v5] Can't start reportportal-api in OpenShift

### ISSUE THREAD

**Issue by b1zzu** [reportportal/reportportal#836]
**Describe the bug**

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
--
  | at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  | at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  | at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  | at java.lang.reflect.Method.invoke(Method.java:498)
  | at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
  | at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
  | at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
  | at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
  | 2020-02-26 16:47:05.661 ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application run failed
  |  
  | java.lang.IllegalStateException: Logback configuration error detected:
  | ERROR in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - openFile(/var/api.log,true) call failed. java.io.FileNotFoundException: /var/api.log (Permission denied)
  | at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:169)
  | at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:80)
  | at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:60)
  | at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:118)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:312)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:287)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:245)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:222)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:127)
  | at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:76)
  | at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:53)
  | at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:345)
  | at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)
  | at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
  | at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
  | at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
  | at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  | at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  | at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  | at java.lang.reflect.Method.invoke(Method.java:498)
  | at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
  | at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
  | at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
  | at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
  |  
  | Caused by: java.lang.IllegalStateException: Logback configuration error detected:
  | ERROR in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - openFile(/var/api.log,true) call failed. java.io.FileNotFoundException: /var/api.log (Permission denied)
  | at org.springframework.boot.logging.logback.LogbackLoggingSystem.loadConfiguration(LogbackLoggingSystem.java:169)
  | at org.springframework.boot.logging.AbstractLoggingSystem.initializeWithConventions(AbstractLoggingSystem.java:80)
  | at org.springframework.boot.logging.AbstractLoggingSystem.initialize(AbstractLoggingSystem.java:60)
  | at org.springframework.boot.logging.logback.LogbackLoggingSystem.initialize(LogbackLoggingSystem.java:118)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.initializeSystem(LoggingApplicationListener.java:312)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.initialize(LoggingApplicationListener.java:287)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEnvironmentPreparedEvent(LoggingApplicationListener.java:245)
  | at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:222)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
  | at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:127)
  | at org.springframework.boot.context.event.EventPublishingRunListener.environmentPrepared(EventPublishingRunListener.java:76)
  | at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:53)
  | at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:345)
  | at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)
  | at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
  | at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
  | at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
  | ... 8 more


```

**To Reproduce**

Install ReportPortal in Kubernetes using Helm

**Comment by HardNorth** [comment]
I install ReportPortal in Kubernetes several times a week and have not seen such issue before.
So you have to specify in details your deployment environment and process if you want to get any help.

**Comment by b1zzu** [comment]
@HardNorth sorry I try to bring more context:

I'm installing it on OpenShift and by default, OpenShift doesn't allow the process of a Pod to run as a user by default, therefore if I don't specify the `runAsUser` and assign a special policy to the default serviceaccount the Pod will start with some random user which doesn't have any privileges to write to `/var/api.log`. 

I don't know how the plain Kubernetes behave, but on OpenShift I've fixed it by manually add to the reportportal-api deployment the `runAsUser: 0` option

**Comment by HardNorth** [comment]
@b1zzu That's better :)
Well, It's actually questionable whether we actually need to save logs inside pods at all. We probably need to introduce a flag to silence any file logging.

**Comment by DzmitryHumianiuk** [comment]
Closed as duplicate in honor of #715
